test/recurly.test.js

```diff
@@ -4,8 +4,20 @@ const assert = require('assert').strict
 const recurly = require('../lib/recurly')
 const Client = require('../lib/recurly/Client')
 const Account = require('../lib/recurly/resources/Account')
+const pkg = require('../package.json')
 
 describe('recurly', () => {
+  describe('Dependencies', () => {
+    it('Should have no runtime dependencies', () => {
+      let numDeps = Object.keys(pkg.dependencies).length
```

Comment:
Do you want to add in `peerDependencies` to this as well?

Reply:
Good point!